README.md

```diff
@@ -22,6 +22,7 @@ separate table to be used at a later date for matching.
 get     /health                     # test endpoint for the application
 get     /deed/<id_>                 # get a deed with an id in the URL
 post    /deed/                       # Create a deed by posting a json object reflecting the schema
+delete  /borrowers/delete/<borrower_id>' # delete a borrower from the borrower table
```

Comment:
Tiny thing but it would look more professional if we had these aligned 
<img width="5" alt="screen shot 2015-12-08 at 16 39 06" src="https://cloud.githubusercontent.com/assets/3599604/11661535/4daa8286-9dca-11e5-9746-0e767ae092c1.png">